src/ui/marker.js

```diff
@@ -297,10 +297,16 @@ export default class Marker extends Evented {
     }
 
     /**
-     * Binds a Popup to the Marker
-     * @param popup an instance of the `Popup` class. If undefined or null, any popup
-     * set on this `Marker` instance is unset
+     * Binds a `Popup` to the `Marker`.
```

Comment:
do you think it'll be better to actually link to #popup here too? I find these internal links very helpful myself.